src/scripts/main.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
-// write code here
+const wholeField = document.querySelector('.container');
+const addRow = document.querySelector('.append-row');
+const delRow = document.querySelector('.remove-row');
+const addColumn = document.querySelector('.append-column');
+const delColumn = document.querySelector('.remove-column');
+const moveField = document.querySelector('tbody');
+
+wholeField.addEventListener('click', (e) => {
+  if (e.target.className === 'append-row button') {
+    const copy = moveField.children[0].cloneNode(true);
+
+    if (moveField.children.length < 10) {
+      moveField.append(copy);
+    }
+
+    if (moveField.children.length > 2) {
+      delRow.disabled = false;
+    }
+
+    if (moveField.children.length >= 10) {
+      addRow.disabled = true;
+    }
```

Comment:
it's better to create separate functions for each action and move all logic there

Reply:
like appendRow, removeRow, appendColumn, etc